xray/core/ops.py

```diff
@@ -465,3 +470,29 @@ def inject_all_ops_and_reduce_methods(cls, priority=50, array_only=True):
             setattr(cls, name, _values_method_wrapper(name))
 
     inject_reduce_methods(cls)
+
+
+def inject_bottleneck_rolling_methods(cls):
+    # standard numpy reduce methods
+    for name in NAN_REDUCE_METHODS:
+        f = getattr(np, name)
+        func = cls._reduce_method(f)
+        func.__name__ = name
+        func.__doc__ = 'todo'
+        setattr(cls, name, func)
+
+    # bottleneck rolling methods
+    if has_bottleneck:
```

Comment:
I'm not sure what that gains us here since they are already defined above using `numpy` functions using the generic `reduce` method.  Everything below implements the `bottleneck` reductions but the behavior should be the same either way.